addon/services/active-storage.js

```diff
@@ -15,7 +15,7 @@ export default class ActiveStorageService extends Service {
     return config['ember-active-storage'] || {};
   }
 
-  upload(file, urlOrCallbacks, callbacks = {}) {
+  upload(file, urlOrCallbacks, callbacks = {}, metadata = {}) {
```

Comment:
If anyone set url in environment they don't send any additional object for that and in that case `callbacks` will become like `callbacksOrMetadata` and in if check after line 30 you need to set `metadata = callbacksOrMetadata`

Got my point? @fsmanuel

Reply:
@sinankeskin I renamed callbacks to options and added documentation for the `metadata` option.